### ISSUE TITLE
PY_SSIZE_T_CLEAN macro must be defined for '#' formats

### ISSUE THREAD

**Issue by kolmogorovpavel** [BrianPugh/game-and-watch-patch#46]
I tried to use script on the Ubuntu with installed Python 3.10.4
and I got error:
```
Traceback (most recent call last):
  File "/home/up/opt/game-and-watch-patch/patch.py", line 178, in <module>
    main()
  File "/home/up/opt/game-and-watch-patch/patch.py", line 104, in main
    device.crypt()  # Decrypt the external firmware
  File "/home/up/opt/game-and-watch-patch/patches/firmware.py", line 492, in crypt
    self.external.crypt(self.internal.key, self.internal.nonce)
  File "/home/up/opt/game-and-watch-patch/patches/firmware.py", line 414, in crypt
    aes = AES.new(key, AES.MODE_ECB)
  File "/home/up/.local/lib/python3.10/site-packages/Crypto/Cipher/AES.py", line 95, in new
    return AESCipher(key, *args, **kwargs)
  File "/home/up/.local/lib/python3.10/site-packages/Crypto/Cipher/AES.py", line 59, in __init__
    blockalgo.BlockAlgo.__init__(self, _AES, key, *args, **kwargs)
  File "/home/up/.local/lib/python3.10/site-packages/Crypto/Cipher/blockalgo.py", line 141, in __init__
    self._cipher = factory.new(key, *args, **kwargs)
SystemError: PY_SSIZE_T_CLEAN macro must be defined for '#' formats
make: *** [Makefile:264: build/internal_flash_patched.bin] Error 1
```
I'm not a specialist in python. Google tells that some define should be used to compile modules. Probably some module is deprecated.

**Comment by BrianPugh** [comment]
It seems like there was a [breaking change in python3.10](https://peps.python.org/pep-0353/).  The most immediate answer is to use an older version of python (3.8 or 3.9).

In the meantime, I'll see if i can mitigate it.

**Comment by BrianPugh** [comment]
See if running the following fixes it; if so i'll update the repo:

```
python -m pip uninstall pycrypto
python -m pip install pycryptodome
```

**Comment by BrianPugh** [comment]
should be addressed in #47
